packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -465,7 +425,7 @@ export const makeInnerVault = (
    * @param {Amount} loss
    * @returns {Amount}
    */
-  const applyDelta = (base, gain, loss) =>
+  const addSubtract = (base, gain, loss) =>
```

Comment:
seems generally useful on `AmountMath` but I don't think we should put it there until there are more instances